lib/foods.js

```diff
@@ -91,17 +41,72 @@ function saveFoodList(whichTable){
     var calories = $(this).children('.food-calories').text();
     foods.push(createNewFood(name, calories));
   });
-  sessionStorage.setItem(storedTableName(whichTable), JSON.stringify(foods));
+  localStorage.setItem(storedTableName(whichTable), JSON.stringify(foods));
```

Comment:
@neight-allen Part of our recent refactor was going from `sessionStorage` to `localStorage`. Should cover one of your suggestions on a previous PR.

FYI: @lucyconklin